CHANGELOG.md

```diff
@@ -2,19 +2,18 @@
 
 * **Breaking change**: `close()` of `DelimiterSyntax` and `LinkSyntax`
   returns multiple nodes instead of single one.
-
-## 6.0.2-dev
-
-* Fix a crash in checkbox lists when mixing checkbox items with
-  non-checkbox items.
+* **Breaking change**: Remove deprecated APIs, including `TagSyntax`,
+  `indicatorForCheckedCheckBox`, and `indicatorForUncheckedCheckBox`.
 * Add a new syntax `BlockHtmlSyntax` to parse HTML blocks.
 * Add a new syntax `DecodeHtmlSyntax` to decode HTML entity and numeric
   character references.
-* Deprecate `BlockTagBlockHtmlSyntax`, `LongBlockHtmlSyntax` and
```

Comment:
These were never exported – and never published. So I think it's okay to drop them.

Reply:
Gah! They were published. But when deprecated, the export was removed. Need to be careful here. We could publish a dot release w/ them deprecated. Or just drop em.

What do you think @srawlins ?

Reply:
We're bumping to 7.0.0 now so you can rebase and continue to drop these deprecated things 😄